src/libs/actions/Policy/Tag.ts

```diff
@@ -719,6 +727,78 @@ function setPolicyTagsRequired(policyID: string, requiresTag: boolean, tagListIn
     API.write(WRITE_COMMANDS.SET_POLICY_TAGS_REQUIRED, parameters, onyxData);
 }
 
+function setPolicyTagGLCode(policyID: string, tagName: string, tagListIndex: number, glCode: string) {
+    const tagListName = Object.keys(allPolicyTags?.[`${ONYXKEYS.COLLECTION.POLICY_TAGS}${policyID}`] ?? {})[tagListIndex];
+    const policyTagToUpdate = allPolicyTags?.[`${ONYXKEYS.COLLECTION.POLICY_TAGS}${policyID}`]?.[tagListName]?.tags?.[tagName] ?? {};
+    const onyxData: OnyxData = {
+        optimisticData: [
+            {
+                onyxMethod: Onyx.METHOD.MERGE,
+                key: `${ONYXKEYS.COLLECTION.POLICY_TAGS}${policyID}`,
+                value: {
+                    [tagListName]: {
+                        tags: {
+                            [tagName]: {
+                                ...policyTagToUpdate,
+                                pendingAction: CONST.RED_BRICK_ROAD_PENDING_ACTION.UPDATE,
+                                pendingFields: {
+                                    glCode: CONST.RED_BRICK_ROAD_PENDING_ACTION.UPDATE,
+                                },
+                                glCode,
```

Comment:
This should go under "GL Code", so "GL Code": glCode

Reply:
@Gonals aren't we're renaming `GL Code` to `glCode`? 

cc: @Beamanator 

https://docs.google.com/document/d/1dkS0mceibjUCMErQ1mWgUHjdHH0Ti_39DvgntRG-sws/edit#bookmark=id.6gfl4go1wes

Reply:
Ah, I don't think we want that. It is "GL Code" in the DB, so I think it should be the same in the Onyx Object

Reply:
Hmmm. I might be wrong, though. let's see what @Beamanator thinks

Reply:
I thinkkk @Gonals is correct, even @cead22 left some comments in the doc, indicating that he also thinks we should store it like it is in the DB - since I am pretty sure that's what we're trying to do "these days" (for least amount of confusion)

Reply:
Can't we rename it in DB? Assuming this is a column rename and isn't used in many places

Reply:
I think it'll be more work to do it in DB and related code. So I'll just rename it on frontend

Reply:
We can't rename it unfortunately, as it is widely used and it would require updating a ton of places